scripts-pipeline/fs.sh

```diff
@@ -68,9 +69,13 @@ this directory already exists. Delete it first and re-run the script, or pass
     fi
 fi
 
-if [ -n "${mask-}" ]; then
+# If mask is given, then mask the img
+if [ -f "${mask}" ]; then
     log "Mask t1"
```

Comment:
We want to check if the $mask argument is non empty and whether the mask file exists. That's why we should need 
`[ -f "${mask}" ]`. Also, I think the dash after mask `"${mask-}" ` was a typo.